### PR TITLE
[SWE-1686][SWE-1687][SWE-1688] AppIcon screens

### PR DESCRIPTION
**Pull request by Olexandr25**
Screenshot 2022-08-03 at 12 38 38
Screenshot 2022-08-03 at 12 42 08
Screenshot 2022-08-03 at 12 42 15

### CODE REVIEW

File: src/app/domains/Iconography/components/AppIcon/AppIcon.js [hunk 1]
```diff
@@ -1,39 +1,36 @@
...
-      <Icon
-        height={height}
-        width={width}
-        src={appIcon}
-        alt="here can be your logo"
-      />
-      <PositionedContainer>
-        <Img src="/icons/logoGrid.svg" alt="ups" />
-      </PositionedContainer>
+      <Box position="absolute">
```
Comment (Anhelina4) [inline src/app/domains/Iconography/components/AppIcon/AppIcon.js:18]: Do you need position here?

File: src/app/domains/Iconography/components/AppIconPhone/AppIconPhone.styled.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 
 const AppIconPhoneStyled = styled.img`
-  position: fixed;
-  width: 851px;
-  height: 1701px;
-  right: 0px;
-  top: -410px;
+  position: absolute;
```
Comment (Anhelina4) [inline src/app/domains/Iconography/components/AppIconPhone/AppIconPhone.styled.js:4]: Do need position & object-fit? Doesn`t it work without them?

File: src/app/domains/Iconography/components/AppIcon/AppIcon.js (hunk 1 above)
Comment (Olexandr25) [reply src/app/domains/Iconography/components/AppIcon/AppIcon.js:18]: Yes, I need here

File: src/app/domains/Iconography/components/AppIconPhone/AppIconPhone.styled.js (hunk 2 above)
Comment (Olexandr25) [reply src/app/domains/Iconography/components/AppIconPhone/AppIconPhone.styled.js:4]: I need them, for making the same, like in figma

**Review by Anhelina4 — APPROVED** [review]

**Review by whoiam460 — APPROVED** [review]

**Pull request merged**